src/zenml/steps/utils.py

```diff
@@ -59,10 +60,7 @@ def get_args(obj: Any) -> Tuple[Any, ...]:
     Returns:
         The args of the annotation.
     """
-    return tuple(
-        pydantic_typing.get_origin(v) or v
-        for v in pydantic_typing.get_args(obj)
-    )
+    return tuple(typing_get_origin(v) or v for v in typing_get_args(obj))
```

Comment:
I remember the pydantic typing covering cases here that `typing_existension` did not. Are we sure this still works?

Reply:
Good catch, this must be a change that slipped away during the first stages. Fixed it.